main.go

```diff
@@ -172,7 +174,49 @@ func filesExists(files []types.FileSecret) error {
 	return nil
 }
 
+func dryProcess(plan types.Plan, prefs InstallPreferences) error {
```

Comment:
This seems to duplicate the work in the process method?

Reply:
@csakshaug thoughts?

Reply:
Will give you feedback later today

Reply:
We could either do a check inside the process for every "task/operation" or have their process where we take out the "task/operation" that make sense to execute in a dry-run. 

Instead call it dry-run perhaps we could focus on just updated the configurations, today the implementation update the yaml files and other scripts for ingress, tls and stack and execute a kubectl apply with dry-run feature.

I have uses this feature a lot for testing and updating the cluster with going through all the files or using the kubectl edit or reset+bootstrap again.